examples/Cargo.toml

```diff
@@ -50,6 +50,14 @@ path = "low-level-api/create_did.rs"
 name = "create_vc"
 path = "low-level-api/create_vc.rs"
 
+[[example]]
+name = "validation_units_vc"
+path = "low-level-api/validation_units_vc.rs"
+
+[[example]]
+name = "validation_units_vp"
+path = "low-level-api/validation_units_vp.rs"
```

Comment:
I think it would also be better to roll them into one. If we can make this PR build on the new resolver instead of the other way round as you suggested it should be easier to make nice examples.